controllers/configmap_controller.go

```diff
@@ -216,10 +230,50 @@ func (r *ConfigMapReconciler) reconcileServices(ctx context.Context, windowsServ
 			kubeTypes.NamespacedName{Namespace: r.watchNamespace, Name: windowsServices.Name}, "Error", err.Error())
 		return nil
 	}
+	r.setValues(node, data)
 	// TODO: actually react to changes to the services ConfigMap
 	return nil
 }
 
+func (r *ConfigMapReconciler) setValues(node *core.Node, data *servicescm.Data) {
```

Comment:
It should be somewhat clear as to what a function is doing from the name/doc.
`setValues` isn't descriptive enough. As this is a method of `ConfigMapReconciler` it implies that this is setting the values for the reconciler itself.